src/core/gui/sidebar/Sidebar.cpp

```diff
@@ -140,6 +140,10 @@ void Sidebar::selectPageNr(size_t page, size_t pdfPage) {
     }
 }
 
+size_t Sidebar::getAmountPages() { return this->pages.size(); }
```

Comment:
The name sounds a bit awkward to me.
```suggestion
size_t Sidebar::getNumberOfPages() { return this->pages.size(); }
```